indolaw-parser/parser_utils.py

```diff
@@ -618,14 +623,26 @@ def insert_perubahan_section_open_quotes(law: List[str]) -> List[str]:
 
             pyperclip.copy(line)
             print('Add open quote in front of line?')
-            print_yes_no()
+            print_yes_no_undo()
             user_input = input()
 
             if user_input == 'y':
                 new_law[i] = OPEN_QUOTE_CHAR + line
+            elif user_input == 'u':
+                if len(i_input) == 0:
+                    print_max_undo()
+                #deletes the OPEN_QUOTE_CHAR if the user input in the index is 'y'
+                elif i_input[-1]['input'] == 'y':
+                    new_law[i_input[-1]['i']] = new_law[i_input[-1]['i']][1:]
+                    i = i_input[-1]['i']
```

Comment:
1) Instead of calling `i_input[-1]['i']` 3x, can we assign it to a separate variable before and give it a nice descriptive name? :)